### PR TITLE
Changed heart icon to like button

### PR DESCRIPTION
**Pull request by jamietsao**
https://github.com/jamietsao/codepath-mapstagram/issues/87

### CODE REVIEW

File: Mapstagram/ViewControllers/ListViewController.swift [hunk 1]
```diff
@@ -130,6 +131,8 @@ extension ListViewController: InstagramActionDelegate {
         if let section = indexPath?.section {
             let media = getMedia()[section]
             media.userHasLiked = true
+            media.likeCount! += 1
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ListViewController.swift]: This is a bug fix where I forgot to increment the like count for the local media object!

File: Mapstagram/Views/LikesTableViewCell.swift [hunk 2]
```diff
@@ -10,8 +10,16 @@ import UIKit
 
 class LikesTableViewCell: UITableViewCell {
     
-    @IBOutlet weak var likeIconImageView: UIImageView!
+    weak var instagramActionDelegate: InstagramActionDelegate!
+    
     @IBOutlet weak var likesCountLabel: UILabel!
+    @IBOutlet weak var likeButton: UIButton! {
```
Comment (jamietsao) [inline Mapstagram/Views/LikesTableViewCell.swift]: Using same button style as the "Follow" button on profile view

File: Mapstagram/Views/LikesTableViewCell.swift [hunk 3]
```diff
@@ -40,4 +53,23 @@ class LikesTableViewCell: UITableViewCell {
         // Configure the view for the selected state
     }
     
+    @IBAction func onLikeButton(sender: AnyObject) {
```
Comment (jamietsao) [inline Mapstagram/Views/LikesTableViewCell.swift]: This is the same code as in the `onDoubleTap` of the `ImageTableViewCell`

**Comment by jamietsao** [comment]
I can merge this before or after @mbatilando's search PR.  Either is fine with me.

**Comment by mbatilando** [comment]
You can merge it now, shouldn't conflict with my stuff

**Pull request merged**